src/python/turicreate/test/test_util.py

```diff
@@ -243,3 +243,35 @@ def test_make_temp_filename(self):
         self.assertTrue(location.startswith(tmp_root))
         self.assertTrue(isinstance(location, str))
         self.assertTrue(os.path.basename(location).startswith(prefix))
+
+    def test_alternate_ssl_path(self):
+        from ..util import _get_temp_file_location
```

Comment:
You are not using either of these two imports.

Reply:
Thank you! I have removed them from the test.